test/blackbox-tests/test-cases/private-package-lib/main.t

```diff
@@ -87,7 +87,30 @@ We want to see mangled names in the dune-package file as well:
     (byte __private__/secret/secret.cma)
     (native __private__/secret/secret.cmxs))
    (native_archives __private__/secret/secret.a)
-     (source (path Secret) (impl (path __private__/secret/secret.ml))))))
+
+
+  $ mkdir prefix
+  $ dune install --prefix $PWD/prefix
```

Comment:
We can do `--dry-run` to save some IO. Or perhaps even just `cat` on the .install file